File: androidx/src/androidTest/java/me/panpf/androidx/test/util/TestWeakAsyncTask2.java
package me.panpf.androidx.test.util;

import android.support.annotation.NonNull;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

import me.panpf.androidx.util.WeakAsyncTask;

public class TestWeakAsyncTask2 extends WeakAsyncTask<WeakAsyncTaskTest, Integer, Integer, Integer> {

    TestWeakAsyncTask2(@NonNull WeakAsyncTaskTest weakAsyncTaskTest) {
        super(weakAsyncTaskTest);
    }

    @Override
    @Nullable
    protected Integer doInBackground(@NotNull WeakAsyncTaskTest weakAsyncTaskTest, @NonNull Integer[] integers) {
        return null;
    }
}

File: androidx-kt/src/androidTest/java/me/panpf/androidxkt/test/util/TestWeakAsyncTask2.kt
package me.panpf.androidxkt.test.util

import me.panpf.androidxkt.util.WeakAsyncTask

class TestWeakAsyncTask2 internal constructor(weakAsyncTaskTest: WeakAsyncTaskTest) : WeakAsyncTask<WeakAsyncTaskTest, Int, Int, Int>(weakAsyncTaskTest) {

    override fun doInBackground(page: WeakAsyncTaskTest, params: Array<Int>): Int? = null
}